### DISCUSSION TITLE
No gui for all Pulsar Plugins

### DISCUSSION THREAD

**Discussion by gatd** [apohl79/audiogridder#713] [Help]
No gui for all Pulsar Audio Plugins vst or vst3
just a bar

**Comment by Oleoproteus** [comment]
we need open GL 2.0, unfortunately my old machine used as a server don't have it....

**Reply by gatd** [reply]
Im using parallels 16 win 10?

**Comment by Oleoproteus** [comment]
I'm using Reaper on Win 10
my Pc is with a recent graphic card but the PC used as a server is old with
an intel integrated chip

[image: image.png]

**Reply by gatd** [reply]
according to open gl extensions viewer  im running version 3.3 open gl ?
